template/v1/Dockerfile

```diff
@@ -141,6 +141,11 @@ ENV OPENSSL_MODULES=/opt/conda/lib64/ossl-modules/
 RUN pip install "krb5>=0.5.1,<0.6" && \
     pip show krb5 | grep Require | xargs -i sh -c '[ $(echo {} | cut -d: -f2 | wc -w) -eq 0 ] '
 
+# Install snowflake-connector-python connector python manually. Snowflake connector python does not yet support
+# platformdirs 4.2, which is supported by conda
+# Remove after https://github.com/snowflakedb/snowflake-connector-python/issues/1876 is fixed
+RUN pip install snowflake-connector-python --no-deps
```

Comment:
See concerns in https://github.com/aws/sagemaker-distribution/pull/220#discussion_r1515160955

Reply:
Hi @dylanraws, can you help elaborate which concerns in [#220 ](https://github.com/aws/sagemaker-distribution/pull/220#discussion_r1515160955) are blocking merge for this PR?